<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>960</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>960</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1200</width>
    <height>960</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>车牌识别</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>车牌图像路径：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="box_imgpath">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>961</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>字符模板目录：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="box_tpdir">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>60</y>
      <width>961</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_imgpath">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>20</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>浏览...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_tpdir">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>60</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>浏览...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_loadimg">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>340</y>
      <width>93</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>加载图片</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_rec">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>340</y>
      <width>93</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>识别车牌</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="box_rec">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>290</y>
      <width>331</width>
      <height>31</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>290</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>识别结果：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_loadtp">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>340</y>
      <width>93</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>加载模板</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupbox1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>721</width>
      <height>841</height>
     </rect>
    </property>
    <property name="title">
     <string>车牌图像</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>681</width>
       <height>791</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="wgt_fig1"/>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupbox2">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>100</y>
      <width>421</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>字符识别</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>381</width>
       <height>121</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="wgt_fig2"/>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>400</y>
      <width>421</width>
      <height>541</height>
     </rect>
    </property>
    <property name="title">
     <string>已加载模板</string>
    </property>
    <widget class="QTableWidget" name="table_tp">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>381</width>
       <height>491</height>
      </rect>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>160</number>
     </attribute>
     <column>
      <property name="text">
       <string>字符</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>模板数</string>
      </property>
     </column>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
